fastlane_bot/events/async_event_update_utils.py

```diff
@@ -544,45 +544,3 @@ def async_update_pools_from_contracts(mgr: Any, current_block: int, logging_path
     mgr.cfg.logger.info(
         f"Async Updating pools from contracts took {(time.time() - start_time):0.4f} seconds"
     )
-
-
-def update_remaining_pools(mgr):
-    remaining_pools = []
-    all_events = [pool[2] for pool in mgr.pools_to_add_from_contracts]
-    for event in all_events:
-        addr = mgr.web3.to_checksum_address(event["address"])
-        ex_name = mgr.exchange_name_from_event(event)
-        if not ex_name:
-            mgr.cfg.logger.warning("[run_async_update_with_retries] ex_name not found from event")
-            continue
-
-        key, key_value = mgr.get_key_and_value(event, addr, ex_name)
-        pool_info = mgr.get_pool_info(key, key_value, ex_name)
-
-        if not pool_info:
-            remaining_pools.append((addr, ex_name, event, key, key_value))
-
-    shuffle(remaining_pools)  # shuffle to avoid repeated immediate failure of the same pool
```

Comment:
The `import shuffle` statement is probably redundant in this file by now

Reply:
fixed